cloud-etl/README.md

```diff
@@ -0,0 +1,121 @@
+![image](../images/confluent-logo-300-2.png)
+
+# Overview
+
+This demo showcases a cloud ETL leveraging all fully-managed services on [Confluent Cloud](https://confluent.cloud?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cloud-etl)
+A source connector reads data from an AWS Kinesis stream into Confluent Cloud, a Confluent KSQL application processes that data, and then a sink connector writes the output data into cloud storage in the provider of your choice.
+
+![image](images/topology.jpg)
+
+This enables you to:
+
+* Build business applications on a full event streaming platform
+* Span multiple cloud providers (AWS, GCP, Azure) and on-prem datacenters
+* Use Kafka to aggregate data in single source of truth
+* Harness the power of [KSQL](https://www.confluent.io/product/ksql/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.cloud-etl)
+
+# End-to-end Streaming ETL
+
+This demo showcases an entire end-to-end cloud ETL deployment, built for 100% cloud services:
+
+* Kinesis source connector: reads from a Kinesis stream and writes the data to a Kafka topic in Confluent Cloud
+* ksqlDB: streaming SQL engine that enables real-time data processing against Kafka
```

Comment:
Nit: I don't think the service we have in Cloud today is technically ksqlDB.

Reply:
@bechols fixed.